app/views/web/teams/show.html.haml

```diff
@@ -0,0 +1,86 @@
+- title @team.title
+- content_for :meta do
+  %meta{ property: 'og:description', content: "#{@team.title} #МИЦ" }
+  %meta{ name:'description', content: "#{@team.title} #МИЦ" }
+.container.account
+  .row
+    .small-5.columns
+      - if @team.address
+        %iframe.team-map{src: google_map_src(@team.address)}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace